### PR TITLE
Update NSS & ca-certificates to 3.44, provide {nss,nspr}-config scripts and man pages

### PR DESCRIPTION
**Pull request by Mno-hime**
Some components need `nspr-config` or `nss-config` scripts (as `.pc` files are not taken into account). Also adding man pages to those from Fedora dist git (with small changes for an easier build).

### CODE REVIEW

File: components/library/mozilla-nss/patches/07-nss-config.patch [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  ld: elf error: file /usr/lib/mps/amd64: elf_begin: I/O error: region read: Is a directory
+
+--- nss-3.44/nss/pkg/pkg-config/nss-config.in	2019-05-10 23:14:18.000000000 +0000
++++ nss-3.44/nss/pkg/pkg-config/nss-config.in.new	2019-05-14 07:16:41.278545068 +0000
+@@ -127,7 +127,7 @@ if test "$echo_cflags" = "yes"; then
+ fi
+ 
+ if test "$echo_libs" = "yes"; then
+-      libdirs="-Wl,-rpath-link,$libdir -L$libdir"
```
Comment (pyhalov) [inline components/library/mozilla-nss/patches/07-nss-config.patch:12]: Should it be -L$libdir -R$libdir ?

Comment (Mno-hime) [reply components/library/mozilla-nss/patches/07-nss-config.patch:12]: Right, thanks.

File: components/library/mozilla-nss/files/nss-config.xml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+    </para>
+
+   
+  </refsection>
+
+  <refsection>
+    <title>Files</title>
+
+    <para><filename>/usr/bin/nss-config</filename></para>
```
Comment (pyhalov) [inline components/library/mozilla-nss/files/nss-config.xml:104]: Should /usr/bin/amd64/nss-config also be mentioned?

Comment (pyhalov) [reply components/library/mozilla-nss/files/nss-config.xml:104]: Otherwise, LGTM

**Pull request merged**